src/sentry/api/authentication.py

```diff
@@ -217,6 +221,31 @@ def authenticate_credentials(self, request: Request, token_str):
         return (token.user, token)
 
 
+class OrgAuthTokenAuthentication(StandardAuthentication):
+    token_name = b"bearer"
+
+    def authenticate_credentials(self, request: Request, token_str):
+        if not token_str.startswith(SENTRY_JWT_PREFIX):
+            return None
+
+        token = None
+        token_hashed = hash_token(token_str)
+
+        try:
+            token = OrgAuthToken.objects.filter(
+                token_hashed=token_hashed, date_deactivated__isnull=True
+            ).get()
+        except OrgAuthToken.DoesNotExist:
+            raise AuthenticationFailed("Invalid org token")
+
+        with configure_scope() as scope:
+            scope.set_tag("api_token_type", self.token_name)
+            scope.set_tag("api_token", token.id)
+            scope.set_tag("api_token_is_org_token", True)
```

Comment:
@mydea can we add the org id as well?

Reply:
sure thing! 👍 I'll add `api_token_org_id`

Reply:
My thought was mirroring the existing scope added here, we may not want to "pollute" the scope as I guess other places may want to set `organization_id` scope? 🤔

Reply:
@mydea it's just more standard that way

Reply:
actually I just checked, I will completely remove this - this is redundant as we already add the organization id & slug somewhere else!